script/build

```diff
@@ -2,18 +2,11 @@
 
 'use strict'
 
-// Run bootstrap first to ensure all the dependencies used later in this script
-// are installed.
-require('./bootstrap')
-
-// Needed so we can require src/module-cache.coffee during generateModuleCache
-require('coffee-script/register')
-require('colors')
-
 const yargs = require('yargs')
```

Comment:
These imports (line 5-12) are not removed but __moved__ after `argv` definition (line 14-24).